fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/api/event/player/PlayerBlockBreakEvents.java

```diff
@@ -55,7 +62,7 @@ private PlayerBlockBreakEvents() { }
 	/**
 	 * Callback after a block is broken.
 	 *
-	 * <p>Only called on a logical server.
+	 * <p>Called on both server and client side.
```

Comment:
I think this is likely a breaking change to the API. Would it be better off being a new client side API?

Reply:
It shouldn’t be breaking for people using this correctly, but I can move it to a new client side API if requested.

Reply:
Existing mods wont be expecting this to be called on the client.

Reply:
Added a new event.